ProyectoSemestral3/Assets/Scripts/PlayerMovement.cs

```diff
@@ -32,6 +32,16 @@ void Update()
             gameObject.transform.forward = move;
         }
 
+        if (Input.GetKeyDown(KeyCode.LeftShift))
+        {
+            speed = 15;
+        }
+
+        if (Input.GetKeyUp(KeyCode.LeftShift))
+        {
+            speed = 5;
```

Comment:
Changing the values like this is considered a magic number, create parameters for them so you can adjust them in the inspector